Gemfile

```diff
@@ -63,7 +63,8 @@ gem "druid-tools"
 gem "dry-types"
 gem "edtf"
 gem "faraday", "~> 2.0"
-gem "globus_client", "~> 0.8"
+# gem "globus_client", "~> 0.8"
+gem "globus_client", git: "https://github.com/sul-dlss/globus_client", branch: "t3021-add-folder-access"
```

Comment:
This will need to be updated before merging, once the globus_client release is cut.

Reply:
Updated now.